tfe/data_source_agent_pool.go

```diff
@@ -21,6 +21,17 @@ func dataSourceTFEAgentPool() *schema.Resource {
 				Type:     schema.TypeString,
 				Optional: true,
 			},
+
+			"organization_scoped": {
+				Type:     schema.TypeBool,
+				Computed: true,
+			},
+
+			"allowed_workspace_ids": {
+				Type:     schema.TypeSet,
+				Computed: true,
+				Elem:     &schema.Schema{Type: schema.TypeString},
+			},
```

Comment:
Should this move to its own data source in order to align with the `tfe_agent_pool_allowed_workspaces` resource?

Reply:
This resource only adds the workspace to the allowed_workspaces attribute of an agent_pool. So this was kept inside the agent_pool data source similar to the team_organization_members